src/SchedulerManager/Travely.SchedulerManager.Repository/Entities/ScheduleInfo.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Travely.SchedulerManager.Repository.Entities
+{
+    public class ScheduleInfo
+    {
+        public ScheduleInfo()
+        {
+            UserSchedules = new HashSet<UserSchedule>();
+        }
+
+        [Key]
+        public int Id { get; set; }
+        public int MessageTemplateId { get; set; }
+        public DateTime ExpirationDate { get; set; }
+        public int RecurseId { get; set; }
+        [Required]
```

Comment:
Why you do not use [fluent API](https://docs.microsoft.com/en-us/ef/core/modeling/)?